packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -407,11 +372,31 @@ export default class ImageEdit implements EditorPlugin {
         htmlData.forEach(data => {
             const element = createElement(data, this.image.ownerDocument);
             if (element) {
-                wrapper.appendChild(element);
+                this.wrapper.appendChild(element);
             }
         });
 
-        this.insertImageWrapper(this.image, wrapper);
+        this.insertImageWrapper(this.image, this.wrapper);
+    }
+
+    private removeVisibilityCssTag() {
+        const doc = this.editor.getDocument();
+        const styleTag = doc.getElementById(STYLE_IMAGE) as HTMLStyleElement;
+        if (styleTag) {
+            doc.head.removeChild(styleTag);
+        }
+    }
+
+    private addVisibilityCssTag() {
+        const doc = this.editor.getDocument();
+        let styleTag = doc.getElementById(STYLE_IMAGE) as HTMLStyleElement;
```

Comment:
Why reuse the same string for style tag and image class?

Also, if there are multiple editor in the same page, there will be duplicated id for style tags then hard to distinguish.

We already have code in editor core to generate STYLE tag and add style rule. Consider reuse that code by adding some editor API or parameter to existing API. The reuse part can be in separate PR, but at least make sure the style element id is unique. You can use editor DIV id.